src/comp_loinc/loinc_builder_steps.py

```diff
@@ -328,6 +340,7 @@ def loinc_term_primary_def(self):
           continue
         loinc_part_id = LoincPartId(part_number)
 
+        setattr(loinc_term, edge_type.name, loinc_part_id)
```

Comment:
## Accidental commit & bug
@ShahimEssaid I believe this is what I was showing you during our meeting; an example of how a lot of the lines below could be refactored with a line like this. I didn't mean to actually commit it.

I believe this caused:
- #87 

The error began with this commit, and caused LOINC term attributes to not be set correctly in the primary part model build. After removing this line, I'm able to build successfully again.

Reply:
Got it. In my rushed comment, I assumed the opposite, that the commit added the setatt() but I now noticed that it's actually removing it.  Yes, that line caused the issue because it probably tried to map an edge from the graph that doesn't belong to the LoincTerm entity (no slot for it).